docs/assets/E2EPerformanceDashboard.json

```diff
@@ -0,0 +1,1592 @@
+{
```

Comment:
Some silly questions: is there any doc to explain the entries in this document? Especially is it possible to introduce stats from insight.py into this dashboard?

Reply:
No Leo we cannot use the stats from our csv/insight.py here

Reply:
Since we use Micrometer to abstract metrics, the tool also supports exporting metrics to Elastic just by enabling some config. Other metrics [providers](https://micrometer.io/docs) can be used but would require an extra jar in the classpath.